veloroutes_voies_vertes/install/sql/veloroutes/60_CONSTRAINT.sql

```diff
@@ -53,11 +63,6 @@ ALTER TABLE ONLY veloroutes.revetement_val
     ADD CONSTRAINT code7 UNIQUE (code);
 
 
--- statut_segment_val code8
-ALTER TABLE ONLY veloroutes.statut_segment_val
-    ADD CONSTRAINT code8 PRIMARY KEY (code);
-
-
 -- etat_avancement_val codeFK
 ALTER TABLE ONLY veloroutes.etat_avancement_val
     ADD CONSTRAINT "codeFK" UNIQUE (code);
```

Comment:
```suggestion
ALTER TABLE ONLY veloroutes.etat_avancement_val
    ADD CONSTRAINT code9 UNIQUE (code);
```

Reply:
rectifications done in the following commit